server/form-pages/apply/risk-management-features/convictedOffences.test.ts

```diff
@@ -0,0 +1,60 @@
+import { itShouldHaveNextValue, itShouldHavePreviousValue } from '../../shared-examples'
+
+import applicationFactory from '../../../testutils/factories/application'
+import personFactory from '../../../testutils/factories/person'
+import ConvictedOffences, { responses } from './convictedOffences'
+import { convertKeyValuePairToRadioItems } from '../../../utils/formUtils'
+
+jest.mock('../../../utils/formUtils')
+
+describe('ConvictedOffences', () => {
+  let application = applicationFactory.build()
+  const person = personFactory.build({ name: 'John Wayne' })
+  application = applicationFactory.build({ person })
```

Comment:
I feel confused by this - any reason not to get rid line 11 can make line 13 `const application = ...`

Reply:
Yeah good point, I copied and pasted this from elsewhere. You're right - 
it doesn't make sense